src/Mvc/Mvc.Razor/src/TagHelpers/UrlResolutionTagHelper.cs

```diff
@@ -178,8 +176,7 @@ protected void ProcessUrlAttribute(string attributeName, TagHelperOutput output)
                 var stringValue = attribute.Value as string;
                 if (stringValue != null)
                 {
-                    string resolvedUrl;
-                    if (TryResolveUrl(stringValue, resolvedUrl: out resolvedUrl))
+                    if (TryResolveUrl(stringValue, resolvedUrl: out string? resolvedUrl))
```

Comment:
does `out var` not work?

Reply:
It's an overloaded method and you have to specify the type for resolution